posts/2021-06-11-kubernetes-operator-with-java-and-quarkus.adoc

```diff
@@ -1,7 +1,9 @@
 = Kubernetes Operator with Java and Quarkus
 
 == The goal of this blog post
-An Operator can be used for various reasons. The most common purposes of operators are managing database clusters, preparing database schemas, restoring backups, etc.
+An Operator can be used for various reasons. The most common purposes of operators are managing database clusters, preparing database schemas, and restoring backups.
+
+I'd avoid using `etc` in your leading sentence.
```

Comment:
This was just a comment, not meant to be included.

Reply:
I updated this part.